src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,37 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final int DATA_INDEX = 0;
+    private static final int NAME_INDEX = 1;
+    private static final int MULTIPLIER_INDEX = 2;
+    private static final int SALARY_INDEX = 3;
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder builder = new StringBuilder();
```

Comment:
do not use such naming, how to guess what it stores?
```suggestion
        StringBuilder report = new StringBuilder();
```